Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/QuestionComponent/question.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import { Button } from "@material-ui/core";
+import Box from "@material-ui/core/Box";
+
+
+
+
+export default function QuestionField(props) {
+
+
+  const useStyles = makeStyles((theme) => ({
+    margin: {
+      margin: theme.spacing(0.5),
+      marginTop: theme.spacing(0),
+      width: 50,
+    },
+  }));
+  
+  
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Box
+        display="flex"
+        flexDirection="row"
+        p={1}
+        m={1}
+        bgcolor="background.paper"
+      >
+        <Box p={1}>
+          <form noValidate autoComplete="off">
+            <TextField
```

Comment:
Fixed this error that some other people made in my Cleanup PR, but for reference you can't have textfields inside of forms I believe.
![13](https://user-images.githubusercontent.com/51762139/96806134-402a1080-13e1-11eb-9120-85840e48a6c6.PNG)